.buildkite/pipeline.yml

```
@@ -223,6 +223,16 @@ steps:
       provider: "gcp"
       machineType: "n1-standard-8"
 
+  - label: "Extended runtime tests"
```

```suggestion
  - label: "Extended runtime leak tests"
```

Someone who didn't read the PR doesn't know what "extended runtime" is actually doing when they see this, let's make it a bit more obvious.